MAUI/Calendar/customizations.md

```diff
@@ -0,0 +1,257 @@
+
+*    **Weekend Dates** – You can customize the weekend dates textstyle and background of the `Calendar` by using the `WeekendDatesTextStyle` and `WeekendDatesBackground` properties of `MonthView`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        Background="PaleGreen">
+            <Calendar:SfCalendar.MonthView>
+                <Calendar:CalendarMonthView TodayBackground="Pink" DisabledDatesBackground="Grey" SpecialDatesBackground="Purple" WeekendDatesBackground="Red"/>
+            </Calendar:SfCalendar.MonthView>
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+        CalendarTextStyle textStyle = new CalendarTextStyle()
+        {
+            TextColor = Colors.Black,
+            FontSize = 12,
+        };
+
+        this.Calendar.MinimumDate = DateTime.Now.AddDays(-15);
+        this.Calendar.MaximumDate = DateTime.Now.AddDays(20);
+        this.Calendar.EnablePastDates = true;
+        this.Calendar.SelectableDayPredicate = (date) =>
+        {
+            if (date.Date == DateTime.Now.AddDays(3).Date || date.Date == DateTime.Now.AddDays(6).Date || date.Date == DateTime.Now.AddDays(-3).Date || date.Date == DateTime.Now.AddDays(-6).Date)
+            {
+                return false;
+            }
+
+            return true;
+        };
+
+        this.Calendar.MonthView.SpecialDates = new List<DateTime>
+        {
+            DateTime.Now.AddDays(2),
+            DateTime.Now.AddDays(-2),
+        };
+
+        this.Calendar.MonthView.WeekendDays = new List<DayOfWeek>
+        {
+            DayOfWeek.Sunday,
+            DayOfWeek.Saturday,
+        };
+
+        this.Calendar.Background = Colors.PaleGreen;
+        this.Calendar.MonthView.TextStyle = textStyle;
+        this.Calendar.MonthView.TodayBackground = Colors.Pink;
+        this.Calendar.MonthView.TodayTextStyle = textStyle;
+        this.Calendar.ShowTrailingAndLeadingDates = true;
+        this.Calendar.MonthView.TrailingLeadingDatesBackground = Colors.Green;
+        this.Calendar.MonthView.TrailingLeadingDatesTextStyle = textStyle;
+        this.Calendar.MonthView.DisabledDatesBackground = Colors.Grey;
+        this.Calendar.MonthView.DisabledDatesTextStyle = textStyle;
+        this.Calendar.MonthView.SpecialDatesBackground = Colors.Purple;
+        this.Calendar.MonthView.SpecialDatesTextStyle = textStyle;
+        this.Calendar.MonthView.WeekendDatesBackground = Colors.Red;
+        this.Calendar.MonthView.WeekendDatesTextStyle = textStyle;
+
+{% endhighlight %}
+{% endtabs %}
+
+![monthview-customization-in-maui-calendar](images/customization/monthview-customization-in-maui-calendar.png)
+
+>**NOTE**
+* The Background color and text style will be applied based on the following order: selectableDayPredicate dates, special dates, disable dates, today date, trailingLeading Dates and weekend dates.
```

Comment:
The Background color and text style will be applied based on the following order: selectableDayPredicate dates, special dates, disable dates, today date, trailingLeading dates, and weekend dates.

Reply:
Content changed